src/oscar_accounts/dashboard/views.py

```diff
@@ -27,6 +28,7 @@ class AccountListView(generic.ListView):
     template_name = 'accounts/dashboard/account_list.html'
     form_class = forms.SearchForm
     description = _("All %s") % names.UNIT_NAME_PLURAL.lower()
+    paginate_by = getattr(settings, 'OSCAR_ACCOUNTS_ACCOUNTS_PER_PAGE', 20)
```

Comment:
I think it would be better to default to using `OSCAR_DASHBOARD_ITEMS_PER_PAGE` instead of defining new settings for each view - this is the setting that Oscar itself uses for all dashboard list views.

Reply:
fixed